lib/src/voip/README.md

```diff
@@ -36,10 +34,11 @@ Sends the `com.famedly.call.member` event to signal an active membership. The fo
             "backend": {
                 "type": "mesh"
             },
-            "call_id": "!qoQQTYnzXOHSdEgqQp:im.staging.famedly.de",
+            "call_id": "",
```

Comment:
Why is that an empty string now?

Reply:
was done to be compatible with element but now it's a thing. Is mostly sugar as it wasn't used anywhere for m.room scoped calls.

m.room scoped calls don't have a callid essentially, callid is only supposed to be used for m.user scoped calls.